src/epub/text/chapter-7.xhtml

```diff
@@ -16,8 +16,8 @@
 			<p>“This,” she explained to Claire, “is where I do my stunt. Watch it. I invented the steps myself. Classical stuff. It’s called the Dream of Psyche.”</p>
-			<p>It was difficult for one who knew her as Claire did to associate Polly Wetherby with anything classical. On the road, in England, when they had been fellow members of the number two company of The Heavenly Waltz, Polly had been remarkable chiefly for a fund of humorous anecdote and a gift, amounting almost to genius, for doing battle with militant landladies. And renewing their intimacy after a hiatus of a little less than a year, Claire had found her unchanged. The moment before the music started Lady Wetherby, ever a warm patron of sport, had been arguing forcefully in favor of the view⁠—opposed, it seemed, by a bunch of boneheaded boobs on certain of the daily papers that the Tennessee Bearcat, though eclipsed by showier rivals over the ten-round route, would be lightweight champion of the world tomorrow if he could only succeed in luring his most prominent rival into the ring for a forty-five-round contest. Claire found herself wondering how her friend could possibly shake off this mood and prepare herself at a moment’s notice to give an artistic rendition of the Dream of Psyche.</p>
-			<p>As a matter of fact, Lady Wetherby did not. Perhaps it was the association of ideas, but it seemed to Claire that the Dream of Psyche, as interpreted in terms of the dance by her friend, was far less like a Dream of Psyche than a troubled nightmare of the Tennessee Bearcat, fallen asleep while brooding on how he should induce the lightweight champion to fight him to a finish. As the performance proceeded she could well believe that it was Polly Wetherby who had invented the steps.</p>
+			<p>It was difficult for one who knew her as Claire did to associate Polly Wetherby with anything classical. On the road, in England, when they had been fellow members of the number two company of The Heavenly Waltz, Polly had been remarkable chiefly for a fund of humorous anecdote and a gift, amounting almost to genius, for doing battle with militant landladies. And renewing their intimacy after a hiatus of a little less than a year, Claire had found her unchanged. The moment before the music started Lady Wetherby, ever a warm patron of sport, had been arguing forcefully in favor of the view⁠—opposed, it seemed, by a bunch of boneheaded boobs on certain of the daily papers⁠—that the Tennessee Bear-cat, though eclipsed by showier rivals over the ten-round route, would be lightweight champion of the world tomorrow if he could only succeed in luring his most prominent rival into the ring for a forty-five-round contest. Claire found herself wondering how her friend could possibly shake off this mood and prepare herself at a moment’s notice to give an artistic rendition of the Dream of Psyche.</p>
```

Comment:
Bear-cat -> Bearcat was a spelling modernization, can you revert those? Thanks!

Reply:
Sure, done.